NuGet.config

```diff
@@ -3,6 +3,9 @@
   <packageSources>
     <clear />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="lab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
+	<add key="net" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
Formatting. 
Why do we need these new feeds? Are all other the projects going to use them as well?